charts/seed-controlplane/charts/etcd/values.yaml

```diff
@@ -44,5 +48,16 @@ etcdResources:
     cpu: "4"
     memory: 30G
 
+brResources:
+  limits:
+    cpu: 500m
+    memory: 2Gi
+  requests: 
+    cpu: 23m 
+    memory: 128Mi
+
 hvpa:
```

Comment:
Is this being used?

Reply:
It is used only if the resources are not adopted from already existing statefulset (basically if statefulsets are newly deployed and not adopted).